tools/_setup_generation/step_getting_started.py

```diff
@@ -11,7 +11,13 @@
 
 class GettingStartedStep(SetupStep):
     def __init__(self):
-        self.navigation = None
+        # {page: (pattern_to_replace, content)}
+        self.DEFAULT_CONTENT = {
```

Comment:
This is where we could define the properties of each Getting Started to automate full generation in the mkdocs yml template.